scripts/install_browsermob_proxy.sh

```diff
@@ -9,27 +18,29 @@ BMP_URL="https://github.com/lightbody/browsermob-proxy/releases/download/$BMP_NA
         echo "Fetching browsermob-proxy package."
         # Use "-L" to follow redirects, which github uses in order to
         # specify the access token for the AWS bucket where it keeps the file
-        curl -L $BMP_URL > /var/tmp/bmp.zip
-        cd /var/tmp
-        unzip /var/tmp/bmp.zip
-        mv /var/tmp/$BMP_NAME /etc/browsermob-proxy
+        curl -L $BMP_URL > $TEMP_BROWSERMOB_DIR/bmp.zip
+        unzip $TEMP_BROWSERMOB_DIR/bmp.zip -d $TEMP_BROWSERMOB_DIR
+        mv $TEMP_BROWSERMOB_DIR/$BMP_NAME $BROWSERMOB_INSTALL_LOC
     else
         echo "browsermob-proxy package found."
     fi
 
     echo "Finishing installation of browsermob-proxy."
     # make sure the packages main script is executable
-    chmod 0755 /etc/browsermob-proxy/bin/browsermob-proxy
+    chmod 0755 $TEMP_BROWSERMOB_DIR/browsermob-proxy/bin
```

Comment:
Ah good point. I've updated it with the latest commit. It will be the install location.